src/domain/registration/utils.ts

```diff
@@ -97,7 +91,12 @@ export const getEditRegistrationWarning = ({
   }
 
   if (!userCanDoAction) {
-    return t('registration.form.editButtonPanel.warningNoRightsToEdit');
+    switch (action) {
+      case REGISTRATION_ACTIONS.CREATE:
+        return t('registration.form.editButtonPanel.warningNoRightsToCreate');
+      default:
+        return t('registration.form.editButtonPanel.warningNoRightsToEdit');
+    }
```

Comment:
I think using switch case here is little bit overkill :)

Reply:
Changed to if-else statement